docs/deployment/image_vector.md

```diff
@@ -13,16 +13,47 @@ images:
   repository: gcr.io/google_containers/pause-amd64
   tag: "3.0"
   version: 1.17.x
+  architecture: "amd64"
```

Comment:
As discussed, let's make this 

```yaml
architectures:
- amd64
- arm64
```

This way, we can remove the wildcard constant. The implicit default (if `len(architectures)==0`) should be `[amd64,arm64]`.